src/client/common/application/encryptedStorage.ts

```diff
@@ -1,49 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { inject, injectable } from 'inversify';
-import { env, ExtensionMode } from 'vscode';
+import { ExtensionMode } from 'vscode';
 import { IS_REMOTE_NATIVE_TEST } from '../../../test/constants';
-import { UseVSCodeNotebookEditorApi } from '../constants';
 import { IExtensionContext } from '../types';
-import { IAuthenticationService, IEncryptedStorage } from './types';
-
-declare const __webpack_require__: typeof require;
-declare const __non_webpack_require__: typeof require;
-function getNodeModule<T>(moduleName: string): T | undefined {
-    const r = typeof __webpack_require__ === 'function' ? __non_webpack_require__ : require;
-    try {
-        return r(`${env.appRoot}/node_modules.asar/${moduleName}`);
-    } catch (err) {
-        // Not in ASAR.
-    }
-    try {
-        return r(`${env.appRoot}/node_modules/${moduleName}`);
-    } catch (err) {
-        // Not available.
-    }
-    return undefined;
-}
-
-// Use it
-type KeyTar = {
-    setPassword(service: string, account: string, password: string): Promise<void>;
-    deletePassword(service: string, account: string): Promise<void>;
-    getPassword(service: string, account: string): Promise<string | undefined>;
-};
-
-const keytar = getNodeModule<KeyTar>('keytar');
+import { IEncryptedStorage } from './types';
```

Comment:
I'm confused. Looks like you're including changes for the secret storage fix too? Can you rebase?

Reply:
Yes, i didn't merge that PR as there were too many other related changes to the proposed API  
& figured i might as well fix them all in this one PR.